skxray/core/correlation.py

```diff
@@ -405,3 +404,338 @@ def auto_corr_scat_factor(lags, beta, relaxation_rate, baseline=1):
     return beta * np.exp(-2 * relaxation_rate * lags) + baseline
 
 
+two_time_results = namedtuple(
```

Comment:
I think that the `results` object can be shared between the one-time and two-time implementations